static/jade/getting-here/index.jade

```diff
@@ -157,23 +157,23 @@ block viewport
 
     p
       ul
-        li 350 CZK (~13 EUR) per night per each person in a double room
+        li 350 CZK (~13 EUR) per night per person in a double room
```

Comment:
Sorry again, I didn't realize what the difference was... I was a bit stoopid yesterday :smile: let's keep "each" there then.